README.md

```diff
@@ -0,0 +1,48 @@
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+Creates a Lambda function with associated role and policies that
+will run the packer-janitor tool to get rid of orphaned Packer
+instances and their associated resources.
+
+Creates the following resources:
+
+* Lambda function
+* IAM role and policies to describe and delete instances, keypairs,
+  and security groups, as well as write messages to Cloudwatch Logs
+* Cloudwatch Logs group
+* Cloudwatch Event to regularly run cleanup job
+
+## Usage
+
+```hcl
+module "packer-janitor-lambda" {
+  source                 = "trussworks/lambda-packerjanitor/aws"
```

Comment:
Do you need `interval_minutes` in here?

Reply:
You don't *need* it -- it has a default of 60 minutes, so you can skip it if you are fine with that (and I suspect most people using this would be okay with that).